projects/observability/src/shared/dashboard/data/graphql/explore/explore-cartesian-data-source.model.ts

```diff
@@ -19,7 +32,51 @@ import { ExploreResult } from './explore-result';
   type: 'explore-cartesian-data-source'
 })
 export class ExploreCartesianDataSourceModel extends GraphQlDataSourceModel<CartesianDataFetcher<ExplorerData>> {
-  public request?: ExploreVisualizationRequest;
+  @ModelProperty({
```

Comment:
I think I am settling on this api now. I plan to create child datasources that would extend from this class and would contain the logic of converting the `CartesianDataFetcher<ExplorerData>` and `CartesianResult<TInterval>` to its specific series configuration info.